specification/migrate/resource-manager/Microsoft.Migrate/stable/2018-02-02/migrate.json

```diff
@@ -1670,9 +1709,183 @@
+        }
+      }
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/deleteSubscriptionNotification": {
+      "delete": {
+        "summary": "Delete all projects in the subscription.",
+        "description": "Delete all projects in the subscription. Deleting non-existent project is a no-operation.",
+        "operationId": "SubscriptionNotification_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/acceptLanguage"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Deleted the projects.",
+            "headers": {
+              "x-ms-request-id": {
+                "type": "string",
+                "description": "Service generated Request ID."
+              }
+            }
+          },
+          "204": {
+            "description": "No content. No project with specified name was found."
+          },
+          "400": {
+            "description": "Bad request. (Invalid subscriptionId)."
+          },
+          "401": {
+            "description": "Not authorized (No access to subscription, resource group, or project)"
+          },
+          "500": {
+            "description": "Internal Server Error."
+          },
+          "503": {
+            "description": "Service Unavailable."
+          }
+        },
+        "x-ms-examples": {
+          "SubscriptionNotification_Delete": {
+            "$ref": "./examples/SubscriptionNotification_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Migrate/locations/{locationName}/assessmentOptions": {
+      "get": {
+        "summary": "Get the assessment options.",
+        "description": "Get the available options for the properties of an assessment.",
+        "operationId": "AssessmentOptions_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/locationName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/acceptLanguage"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Returns assessment options.",
+            "schema": {
+              "$ref": "#/definitions/AssessmentOptionsResultList"
+            }
+          },
+          "401": {
+            "description": "Not authorized (No access to subscription)."
+          },
+          "404": {
+            "description": "Not found (Invalid subscriptionId)."
+          },
+          "500": {
+            "description": "Internal Server Error."
+          },
+          "503": {
+            "description": "Service Unavailable."
+          }
+        },
+        "x-ms-pageable": {
```

Comment:
I don't think you can make this operation pageable - the list response has two separate list parameters.
When I try to generate a Python SDK it fails in attempting to make this operation pageable.

Reply:
Is there a way, we can suppress this warning for now? We will make the response pageable in the next API version change.

Reply:
Hi @vivekkothakota - this is currently an error, so it will need to be modified. When you say it will be supported in a subsequent API version, do you mean that the response will have a `nextLink`? Or do you mean that the model will be restructured to have a single iterable array?

The main reason it is failing to generate is because you need to specify the name of the array you wish to be iterated in the paged response. The response model currently defines two different arrays.
Some languages do not support multiple arrays in the response - so whichever array you select to be the iterating array, the other array will be lost during deserliazation. If you still want to do this, you need to define `itemName` as described the [x-ms-pageable docs](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable).

Alternatively, simply remove x-ms-ageable for now, and both arrays in the response will be deserilized as a normal model.

Reply:
thank you. remove x-ms-pageable for now..